ingestjob/logger.go

```diff
@@ -256,3 +257,51 @@ func generateID(log *Log) (string, error) {
 	}
 	return docID, nil
 }
+
+// WriteTask ...
+func (s *Logger) WriteTask(log *TaskLog) error {
+	if log.Connector == "" || len(log.Configuration) == 0 || log.CreatedAt.IsZero() {
+		return fmt.Errorf("error: log connector, configuration and created at are all required")
+	}
+
+	docID, err := generateTaskID(log)
```

Comment:
@aultron as per ticket requirement, task id should be ECS task id.because it will be used to query running ECS tasks.